app/decorators/ncr/work_order_decorator.rb

```diff
@@ -0,0 +1,28 @@
+module Ncr
+  class WorkOrderDecorator < Draper::Decorator
+    delegate_all
+
+    EMERGENCY_APPROVER_EMAIL = 'Emergency - Verbal Approval'
+    NO_APPROVER_FOUND        = 'No Approver Found'
```

Comment:
I trained myself to appreciate vertical alignment when writing Perl. It's in the Perl Best Practices book (https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=perl+best+practices+vertical+align+assignment).

That said, I tend to use it only when I think it makes the code more legible, as when setting constants (e.g.). I'm not wedded to it, just have a habit of it. If the consensus is not to do it, I will try and un-train myself.